src/main/java/Duke.java

```diff
@@ -1,10 +1,128 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    private static void initialiseWelcomeMessage() {
+        String logo
+                = "******************************************************\n"
+                + "*                 Systems: [Online]                  *\n"
+                + "*                Protocol: [Dominion]                *\n"
+                + "*                    Race: [Terran]                  *\n"
+                + "******************************************************\n";
+
+        String greeting
+                = "______________________________________________________\n"
+                + "Systems Accessed...\n"
+                + "Decrypting Overwrite...\n"
+                + "Welcome Commander, can I be of assistance, commander?\n"
+                + "______________________________________________________\n";
+
+        System.out.println(logo);
+        System.out.println(greeting);
+    }
+
+    private static void printGoodbyeMessage() {
+        String goodbye
+                = "______________________________________________________\n"
+                + "Good Bye Commander.\n"
+                + "______________________________________________________\n";
+
+        System.out.println(goodbye);
+    }
+
+    private static String getCommand(String userInput) {
+        String[] inputArray = userInput.split(" ");
+        String command = inputArray[0];
+        return command.toUpperCase();
+    }
+
+    private static void executeCommand(String command, String userInput) {
+        String errand;
+        String timestamp;
+        String timestampHeader;
+
+        switch(command) {
+        case "TODO":
+            errand = getSubstring(userInput);
+            Task todo = new Todo(errand);
+            todo.addToTaskList(errand, null); // Todo has no timestamp
+            break;
+        case "EVENT":
+            errand = getErrand(userInput);
+            timestamp = getTimestamp(userInput);
+            timestampHeader = getTimestampHeader(userInput);
+            Task event = new Event(errand, timestamp, timestampHeader);
+            event.addToTaskList(errand, timestamp);
```

Comment:
I like how you refactor your codes to increase the readability.
The code is neat and really easy to understand.
This can be seen in other parts of your code as well.